app/components/MemberListItem/styles.css

```diff
@@ -1,10 +1,18 @@
-.memberListItem { /* stylelint-disable */
+.memberListItem {
   display: flex;
-  justify-content: center;
+  justify-content: left;
```

Comment:
I changed this because it looked kinda weird with centering when the left side had more padding than the right one.